src/components/Content/Pursuance/PursuanceMenu.js

```diff
@@ -60,13 +61,20 @@ const PursuanceMenu = ({ currentPursuanceId, location }) => {
           icon={<Group size={28} />}
         />
         <PursuanceMenuItem
-          className="pursuance-bottom-btn"
           pursuanceId={currentPursuanceId}
           location={location}
           label='Progress'
           action='progress'
           icon={<Rocket size={28} />}
         />
+        <PursuanceMenuItem
+          className="pursuance-bottom-btn"
+          pursuanceId={currentPursuanceId}
+          location={location}
+          label='Map'
+          action='map'
```

Comment:
@colinfruit Please call this "Universe" in the UI and the URL; the rest can stay.

Reply:
👍